README.markdown

```diff
@@ -22,13 +22,14 @@ Web posts are send using the HTTParty gem. The message is sent as the request bo
 
     messenger http://example.com "Message"
 
+Slack
+---
 
-Campfire
```

Comment:
Also, campfire is mentioned higher up in the intro of the Readme. If you're going to remove it, get it there too. Otherwise I'd leave all of the examples and mention Slack in addition to Campfire, rather than removing it.